### PR TITLE
Kitchensink / Fields improvements

### LINKED ISSUE #294: Remove ` 'define' : fielddefinition`, because superfluous

### PR DESCRIPTION
**Pull request by bobdenotter**
Fixes #294

### CODE REVIEW

File: templates/pages/placeholder.html.twig [hunk 1]
```diff
@@ -1,28 +1,79 @@
...
+    </section>
+
+
+    <section class="fields">
+        <h1>Fields</h1>
+
+        <form method="post" id="editor">
+
+        {# a field with _no_ options, all defaults/fallbacks #}
+        {% include ['_partials/fields/generic.html.twig'] %}
```
Comment (JarJak) [inline templates/pages/placeholder.html.twig]: remove square brackets

File: templates/pages/placeholder.html.twig [hunk 2]
```diff
@@ -1,28 +1,79 @@
...
+
+        {# a field with passed in options #}
+        {% include ['_partials/fields/generic.html.twig'] with {
```
Comment (JarJak) [inline templates/pages/placeholder.html.twig]: remove square brackets

File: templates/_partials/fields/_base.html.twig [hunk 3]
```diff
@@ -4,16 +4,16 @@
 {% set type = field.type|default() %}
```
Comment (JarJak) [inline templates/_partials/fields/_base.html.twig]: >Also let's add a comment on the top of this template which variables are required and which are optional.

How about that?

File: templates/pages/placeholder.html.twig (hunk 1 above)
Comment (bobdenotter) [reply templates/pages/placeholder.html.twig]: done

File: templates/pages/placeholder.html.twig (hunk 2 above)
Comment (bobdenotter) [reply templates/pages/placeholder.html.twig]: done

File: templates/_partials/fields/_base.html.twig (hunk 3 above)
Comment (bobdenotter) [reply templates/_partials/fields/_base.html.twig]: Added the description to #293

File: templates/pages/placeholder.html.twig (hunk 2 above)
Comment (JarJak) [reply templates/pages/placeholder.html.twig]: I can still see them, did you forget to push? :)

Comment (bobdenotter) [reply templates/pages/placeholder.html.twig]: Crap! I did! Pushed now.

**Pull request merged**